pkg/controllers/termination/terminate.go

```diff
@@ -65,15 +65,15 @@ func (t *Terminator) drain(ctx context.Context, node *v1.Node) (bool, error) {
 
 	// 2. Separate pods as non-critical and critical
 	// https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown
-	for _, pod := range pods {
-		if val := pod.Annotations[v1alpha5.DoNotEvictPodAnnotationKey]; val == "true" {
-			logging.FromContext(ctx).Debugf("Unable to drain node, pod %s has do-not-evict annotation", pod.Name)
+	for _, nodePod := range pods {
```

Comment:
`chore: variable name (pod) collides with package name` which package?

Reply:
Consider using `p` to avoid this conflict.